core/templates/dev/head/pages/notifications-dashboard-page/notifications-dashboard-page.mainpage.html

```diff
@@ -1,31 +1,68 @@
-{% extends 'dist/base.html' %}
+<!DOCTYPE html>
+<html ng-app="oppia" lang="<[currentLang]>" ng-controller="Base" itemscope itemtype="http://schema.org/Organization">
 
-{% block maintitle %}
-  Notifications - Oppia
-{% endblock maintitle %}
+  <head>
+    @require('../../base_components/header.html', {"title": "Notifications - Oppia"})
+  </head>
 
-{% block navbar_breadcrumb %}
-  <ul class="nav navbar-nav oppia-navbar-breadcrumb">
-    <li>
-      <span class="oppia-navbar-breadcrumb-separator"></span>
-      Notifications
-    </li>
-  </ul>
-{% endblock navbar_breadcrumb %}
+  <body>
+    <base-content>
+      <navbar-breadcrumb>
+        <ul class="nav navbar-nav oppia-navbar-breadcrumb">
+          <li>
+            <span class="oppia-navbar-breadcrumb-separator"></span>
+            Notifications
+          </li>
+        </ul>
+      </navbar-breadcrumb>
 
-{% block content %}
-  <notifications-dashboard-page></notifications-dashboard-page>
-{% endblock %}
+      <content>
+        <notifications-dashboard-page></notifications-dashboard-page>
+      </content>
 
-{% block footer_js %}
-  {{ super() }}
-  <!-- This code is used for inserting webpack bundles
-     https://github.com/jantimon/html-webpack-plugin#writing-your-own-templates -->
-  <% for (var chunk in htmlWebpackPlugin.files.js) { %>
-    <% if (webpackConfig.mode == 'production') { %>
-      <script src="/build/templates/head/dist/<%= htmlWebpackPlugin.files.js[chunk] %>"></script>
-    <% } else { %>
-      <script src="/dist/<%= htmlWebpackPlugin.files.js[chunk] %>"></script>
-    <% } %>
-  <% } %>
-{% endblock footer_js %}
+      <page-footer></page-footer>
+    </base-content>
+
+    <style>
+      .oppia-page-heading-subtext {
+        font-size: smaller;
+        opacity: 0.7;
+      }
+      .oppia-notifications-dashboard-card {
+        padding: 15px 40px;
+      }
+      .oppia-dashboard-row {
+        background-color: #fff;
+        cursor: pointer;
+        margin-bottom: -5px;
+        overflow: auto;
+        position: relative;
+      }
+      .oppia-dashboard-row:hover {
+        background: #eee;
+      }
+      .oppia-dashboard-row-recent {
+        background: #fff4ca;
+      }
+      .oppia-dashboard-row-recent:hover {
+        background: #ffe279;
+      }
+      .oppia-notification-content {
+        color: black;
+        display: inline-block;
+        float: left;
+      }
+      .oppia-notification-last-updated {
+        float: right;
+      }
+      @media (max-width: 500px) {
+        .oppia-notification-content,
+        .oppia-notification-last-updated {
+          display: inline;
+        }
+      }
+    </style>
```

Comment:
The `<style>` part is already in [notifications-dashboard-page.directive.html](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/pages/notifications-dashboard-page/notifications-dashboard-page.directive.html)